src/routes/safe/components/Transactions/TxList/TxInfoCreation.tsx

```diff
@@ -26,57 +27,68 @@ export const TxInfoCreation = ({ transaction }: { transaction: Transaction }): R
 
   return (
     <TxDetailsContainer>
-      <div className="tx-summary">
-        <TxDataRow title="Transaction hash:" value={txInfo.transactionHash} inlineType="hash" />
-        <TxDataRow title="Created:" value={formatDateTime(timestamp)} />
-        <TxDataRow title="Creator:">
-          <PrefixedEthHashInfo
-            textSize="xl"
-            hash={txInfo.creator.value}
-            showCopyBtn
-            explorerUrl={getExplorerInfo(txInfo.creator.value)}
-            name={creator.name || undefined}
-            customAvatar={creator.logoUri || undefined}
-            showAvatar
-          />
-        </TxDataRow>
-        <TxDataRow title="Factory:">
-          {txInfo.factory ? (
-            <PrefixedEthHashInfo
-              textSize="xl"
-              hash={txInfo.factory.value}
-              showCopyBtn
-              explorerUrl={getExplorerInfo(txInfo.factory.value)}
-              name={factory?.name || undefined}
-              customAvatar={factory?.logoUri || undefined}
-              showAvatar
-            />
-          ) : (
-            <Text size="xl" as="span">
-              {NOT_AVAILABLE}
+      <div className="tx-creation">
+        <div>
+          <StyledTxCreationAddress>
+            <Text size="xl" strong>
+              Creator:
```

Comment:
You could refactor this into a factory-esque function, e.g.

```ts
  const getCreationEntity = (title: string, entity: Transaction['creator' | 'factory' | 'implementation']): ReactElement
```

Reply:
That would save some repetition. Do we have an example of it?

Reply:
Just generate the JSX as suggested.